src/components/Theme/README.md

```diff
@@ -348,14 +348,18 @@ export default {
 				fonts: {
 					baseSize: 16,
 					sizeScale: 1.15,
-					title: {
+					heading: {
```

Comment:
How are we supporting the fourth option for `headline`?

Reply:
maker does not use any headlines in any components

Reply:
heading applies to both title and headline patterns

Reply:
https://github.com/square/maker/pull/311/files#diff-b34aee85b50fd2933be427ceddd7edcdb6c5b6eec78d8ae56cbe88838d98df68R83-R84

and

https://github.com/square/maker/pull/311/files#diff-b34aee85b50fd2933be427ceddd7edcdb6c5b6eec78d8ae56cbe88838d98df68R90-R91